### ISSUE TITLE
Upload data from azurehound hangs with no error

### ISSUE THREAD

**Issue by rpetersson** [BloodHoundAD/AzureHound#76]
Trying to identify what the problem is with a dataset collected with azure hound for a specific tenant. It is working fine for different tenants so must be something tenant-specific.

Requesting some kind of error message from Bloodhound GUI instead of current behavior where it just freezes for eternity.

**Comment by StephenHinck** [comment]
Good morning - sorry to hear you're having trouble with this issue. Can you please provide more information about the state of your environment?

- What version of BloodHound CE are you running?
- What does the Finished Ingest Log look like?
- Are there any errors in the API logs?

Happy to help, but we need some additional information to be able to assist.

**Comment by rpetersson** [comment]
11.5.0 - darwin

But where can i find the logs on MacOS?

**Comment by StephenHinck** [comment]
Hi @rpetersson ,

You can find the logs via the `docker logs` command while the environment is running.

I also would still need answers to these questions:

- What version of BloodHound CE are you running?
- What does the Finished Ingest Log look like?

**Comment by 0luc** [comment]
I have the same issue, tried it on different BloodHound installs/environments, but it seems to be a problem with my .json output. Currently trying on WSL.

Things I tried:
- Generated a AzureHound output for a second time (months later).
- Tried to splice the file, but it then hangs only on specific parts of the original file. 
- Tried bloodhound-import module.
- Searched logging, but cannot find any logging about it, nor in Bloodhound or Neo4j logging. 
- Replaced some TenantID's which did not seem my own, read something about that. But import still hung on the same moment.

Any pointers for logging regarding the file uploads? 

Currently doing this on the latest version of AzureHound (v2.2.1) and BloodHound (v4.3.1)

Edit: just noticed that bloodhound version is not the latest version...was replaced with: https://github.com/SpecterOps/BloodHound It works fine with CE :)
